src/components/Pages/Admin/View.vue

```diff
@@ -216,16 +234,57 @@ export default {
 
                 this.GeogebraViews = new GeogebraViews(this.groupsInClass, {
                     log: this.$log,
-                    width: this.$refs.ibox_content.clientWidth - 60,
+                    width: 400, // this.$refs.ibox_content.clientWidth - 60,
+                    height: 400,
                 });
 
                 this.GeogebraViews.inject();
             }
         },
+
+        async mergeViews(checkedGroupIds) {
+            this.showMergedApplet = true;
+            this.liveMergeSwitch = false;
+
+            this.selectedGroups = await this.findGroups({ query: { _id: checkedGroupIds } });
+
+            this.$log.debug('checkedGroups', checkedGroupIds);
+            this.$log.debug('this.selectedGroups', this.selectedGroups);
+
+            if (this.selectedGroups && this.selectedGroups.length) {
+                /* await api.service('users').patch(this.user.username, {
+                    workshops: this.selectedGroups.map(g => g._id),
+                }); */
+
+                this.$log.debug('this.showMenuBar', this.showMenuBar);
+                this.$log.debug('this.$refs.ibox_content.clientWidth',
+                    this.$refs.ibox_content.clientWidth);
+
+                this.GeogebraViews.mergeViews(
+                    this.selectedGroups.map(g => g._id),
+                    { // params for geogebra
+                        showMenubar: this.showMenuBar,
+                        width: (this.$refs.ibox_content.clientWidth - 60),
+                        height: (this.$refs.ibox_content.clientWidth - 60) * 2 / 3,
```

Comment:
This is magic as hell. Try to provide some explanation why you do those calculations in the comment.

Also: refactor out `this.$refs.ibox_content.clientWidth - 60`